packages/kit/src/core/dev/index.js

```diff
@@ -325,114 +333,119 @@ async function create_handler(vite, config, dir, cwd, manifest) {
 				const host = /** @type {string} */ (config.kit.host ||
 					req.headers[config.kit.hostHeader || 'host']);
 
-				const rendered = await respond(
-					{
-						headers: /** @type {import('types/helper').Headers} */ (req.headers),
-						method: req.method,
-						host,
-						path: parsed.pathname,
-						query: parsed.searchParams,
-						rawBody: body
-					},
-					{
-						amp: config.kit.amp,
-						dev: true,
-						entry: {
-							file: `/${SVELTE_KIT}/dev/runtime/internal/start.js`,
-							css: [],
-							js: []
-						},
-						floc: config.kit.floc,
-						get_stack: (error) => {
-							vite.ssrFixStacktrace(error);
-							return error.stack;
-						},
-						handle_error: /** @param {Error & {frame?: string}} error */ (error) => {
-							vite.ssrFixStacktrace(error);
-							console.error(colors.bold().red(error.message));
-							if (error.frame) {
-								console.error(colors.gray(error.frame));
-							}
-							if (error.stack) {
-								console.error(colors.gray(error.stack));
-							}
-						},
-						hooks: {
-							getSession: hooks.getSession || (() => ({})),
-							handle: hooks.handle || (({ request, resolve }) => resolve(request)),
-							serverFetch: hooks.serverFetch || fetch
+				const rendered =
+					parsed.pathname.startsWith(config.kit.paths.base) &&
```

Comment:
I wonder if you could create a variable for `parsed.pathname.startsWith(config.kit.paths.base)` to avoid prettier indenting all the subsequent lines

Reply:
it won't make any difference, it could be `x &&` and prettier would still put them on separate lines

Reply:
Ah. You could change it like this to avoid the extra indentation:
```
				if (!parsed.pathname.startsWith(config.kit.paths.base)) {
					res.statusCode = 404;
					res.end('Not found');
					return;
				}
				const rendered = await respond(
```

Reply:
we could, i just don't love the duplication of 

```
res.statusCode = 404;
res.end('Not found');
```

maybe it's the lesser of two evils. not sure

Reply:
could refactor that out:
```
function not_found(res) {
  res.statusCode = 404;
  res.end('Not found');
}
if (!parsed.pathname.startsWith(config.kit.paths.base)) {
  return not_found(res);
}
```